wiki/docs/toolchain.md

```diff
@@ -1,17 +1,47 @@
 Toolchain
 ---
 
-To build Mimiker you will need a toolchain, i.e. compiler, linker, ELF tools and
-debugger. The default option is to choose quite recent _LLVM toolchain_ (check
-version in [tools.mk][6]), i.e. `clang`, `lld`, `llvm` from [apt.llvm.org][7],
-and [gdb-multiarch][8]. However you'll need to reconfigure [launch][9] script to
-use same `gdb-multiarch` binary for each target architecture, look for `gdb`
-section and find `binary` key. 
+To build and run Mimiker you will need a toolchain, i.e. compiler, linker, ELF
+tools emulator and debugger.
 
-You need to install patched version of [QEmu][11] as well. Our version solves
-several issues not patched in the mainstream version – please refer to our list
-of [patches][10]. Prebuild package for Debian x86-64 can be found [here][5].
+## Building tools
+The default option is to choose quite recent _LLVM toolchain_ (check version in
+[tools.mk][6]), i.e. `clang`, `lld`, `llvm` from [apt.llvm.org][7].
 
+List of building tools:
+- make
+- clang
+- llvm
+- lld
+- gperf
+- byacc
+- cpio
```

Comment:
you also need gzip (arm64 build uses compressed kernel image), sh (used by various scripts), awk with support for nawk extensions (used by makesyscalls.sh and genassym.sh scripts), sed (used by gen(t)hash script used by libterminfo) and find (findutils) (used by make rules)

note: in addition to that default target for sys/ builds cscope, etags and tags databases, so you need to modify sys/Makefile to remove them from dependencies

Reply:
also dtc (device-tree-compiler package on Debian)

Reply:
Why list all dependencies instead of pointing the user to `Dockerfile` which has all required dependencies for Mimiker to build?

Reply:
> also dtc (device-tree-compiler package on Debian)

it's not needed for build, it's loaded by qemu when you start vm, I commented about that under running tools section